rocketpy/Flight.py

```diff
@@ -1207,13 +1207,12 @@ def __init_equations_of_motion(self):
 
     @cached_property
     def effective_1rl(self):
-        nozzle = (
-            self.rocket.motor_position - self.rocket.center_of_dry_mass_position
-        ) * self.rocket._csys  # Kinda works for single nozzle
+        nozzle = self.rocket.motor_position
```

Comment:
I don't think this is correct. The new behavior of the motor position parameter is not anymore bound to the nozzle. The nozzle position is specified by the motor parameter `nozzle_position`, therefore I believe it should be:

```py
# Example only, not checked throughout
nozzle = self.rocket.motor_position + self.rocket.motor.nozzle_position
```

Furthermore, I am not sure how the coordinate system orientation is handled in your additions, remember that the motor also has an orientation.

Reply:
Ah right! I forgot about the motor csys

Anyway, I think what you suggested is not right, it should be:

```
nozzle = self.rocket.motor_position + self.rocket._csys*self.rocket.motor.nozzle_position
```

I will add it to the code right away

Also, this makes me think that we need some tests with the Motor and Rocket classes that asserts these position related attributes

Reply:
Actually, there might be a bigger misunderstanding here:

![image](https://github.com/RocketPy-Team/RocketPy/assets/69485049/84ba421f-c81f-4a87-ab59-1d9c584aee64)

The documentation says that the `motor_position` is the position of the nozzle in relation to the rocket's coordinate system.

Was this meant to be different?

If so, why? To me it makes a lot more sense to input the position of the nozzle outlet in relation to the rocket csys

Reply:
I believe that the parameter `position` should be position of the `motor` system "zero" (that might be the nozzle) in the Rocket reference frame.

As a related sidenote that may be corrected in another PR, see the `add_tank` method in `LiquidMotor`, the documentation there seems to be also out of date: it says position from the nozzle tip, however this is not what is assumed in code regarding the tanks.